politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -4,6 +4,24 @@ import (
 	"github.com/decred/politeia/politeiawww/api/v1"
 )
 
+// Help message displayed for the command 'politeiawwwcli help edituser'
+var EditUserCmdHelpMsg = `edituser "userid" "action" "reason"
+
+Edit the details for the given user id. Requires admin privileges.
+
+Valid bit flags are:
+'0' - User does not receive email notifications   
+'1' - User receives email notificaions
```

Comment:
This part is not accurate.  The valid vote bits can be found here:

https://github.com/decred/politeia/blob/f13e494415aa6553be9d965e5b05a1fbc8475284/politeiawww/api/v1/v1.go#L223-L231

Reply:
Took a stab at descriptions based on variable names. Also changed the command description because not all vote bits require admin privileges (or do they?). Have this now:

```
Edit the details for the given user id. 

Valid bit flags are:
'0' - Notification email when status of user's proposal changes
'1' - Notification email when voting has started on user's proposal
'2' - Notification email when a proposal is vetted
'3' - Notification email when a proposal is edited
'4' - Notification email when voting on a proposal has started
'5' - Notification email when a new proposal is submitted (admin only)
'6' - Notification email when a user authorizes vote on proposal (admin only)
'7' - Notification email when a comment is made on a user's proposal
'8' - Notification email when a comment is made on a user's comment
```

Reply:
We really should have human readable options for this, but that's for a different PR.  For now, the valid vote bits can be specified like this.

```
Valid bit flags are:
'1 << 0' - Notification email when status of user's proposal changes
'1 << 1' - Notification email when voting has started on user's proposal
'1 << 2' - Notification email when a proposal is vetted
'1 << 3' - Notification email when a proposal is edited
'1 << 4' - Notification email when voting on a proposal has started
'1 << 5' - Notification email when a new proposal is submitted (admin only)
'1 << 6' - Notification email when a user authorizes vote on proposal (admin only)
'1 << 7' - Notification email when a comment is made on a user's proposal
'1 << 8' - Notification email when a comment is made on a user's comment
```